blockbook.go

```diff
@@ -650,3 +655,38 @@ func computeFeeStats(stopCompute chan os.Signal, blockFrom, blockTo int, db *db.
 	glog.Info("computeFeeStats finished in ", time.Since(start))
 	return err
 }
+
+func initFiatRatesDownloader(db *db.RocksDB, configfile string) {
+	data, err := ioutil.ReadFile(configfile)
+	if err != nil {
+		glog.Errorf("Error reading file %v, %v", configfile, err)
+		return
+	}
+
+	var config struct {
+		FiatRates       string `json:"fiat_rates"`
+		FiatRatesParams string `json:"fiat_rates_params"`
+	}
+
+	err = json.Unmarshal(data, &config)
+	if err != nil {
+		glog.Errorf("Error parsing config file %v, %v", configfile, err)
+		return
+	}
+
+	if config.FiatRates == "" || config.FiatRatesParams == "" {
```

Comment:
I would regard empty FiatRates not as error but simply not enabled functionality, return info only

Reply:
Right